src/main/java/frc/robot/subsystems/swerve/SwerveSubsystem.java

```diff
@@ -31,6 +31,19 @@ public class SwerveSubsystem extends SubsystemBase {
     public static final double MAX_ACCEL = 3; // Max robot tangential acceleration, in m/s^2
     public static final double MAX_OMEGA = Math.toRadians(30); // Max robot angular velocity, in rads/s
 
+    private final Timer lockTimer;
+    private static final double LOCK_TIMEOUT_SECONDS = 1.0; // The elapsed idle time to wait before locking
+    private static final boolean LOCK_TIMEOUT_ENABLE = true;
```

Comment:
possible misnomer? seems to be whether to lock at all

Reply:
Perhaps; might be better as `LOCKING_ENABLE` then?